docs/reference/r-file-system-path.md

```diff
@@ -0,0 +1,36 @@
+# File System Path
+
+In the following table, we summarize some of the most relevant file path used in 
+the Kafka and ZooKeeper charms.
+
+## Kafka 
+
+| Path                                           | Description                                                                                                                                                                                           | Permission                                    |
+|------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|
+| /snap/charmed-kafka/current/opt/kafka          | Binary files for the Charmed Kafka distribution. Note that this is a readonly squashfs file system.                                                                                                   | (read-only)                                   | 
+| /snap/charmed-kafka/current/opt/kafka/bin/*.sh | General bash scripts to provide helpers and utilities for managing and interacting with Kafka.                                                                                                        | (read-only)                                   |
+| /var/snap/charmed-kafka/current/etc/kafka/     | Configuration files used by Kafka daemon process. These files are generally written and managed by the charm.                                                                                         | (owned by `snap_daemon`, managed by `charm`)  |
+| /var/snap/charmed-kafka/common/var/log/kafka/  | Application Logging files generated by the Kafka daemon process. These files are written by the workload, but they may be read by other components to provide monitoring (e.g. grafana and/or charm). | (owned and managed by `snap_daemon`)          |
+| /var/snap/charmed-kafka/common/var/lib/kafka/  | Raw data stored persistently by Kafka during its operations. The files are written and managed by Kafka only.                                                                                         | (owned and managed by `snap_daemon`)          |
+
+
+External storage is used for storing persistent raw data, and it is  
+mounted at `/var/snap/charmed-kafka/common/var/lib/kafka/<storage-id>`, 
+where `<storage-id>` is a progressive number allowing multiple storage units
+for providing both horizontal scalability and provide IO parallelization to 
+enhance throughput. 
```

Comment:
This is a loong sentence for a brain to process, and I found it hard to understand what `<storage-id>` actually is.
Is it a unique ID that gets generated automatically when an external storage unit is mounted? (and the advantages of this are that it allows for multiple storage units, which provides scalability and parallelization etc etc?)

I would provide a simple definition of `<storage-id>` and separate the last part into its own sentence, like "Multiple storage units provides horizontal scalability and provides IO parallelization to enhance throughput."